src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -31,12 +31,30 @@
             jQuery('.gridItem').click(function () {
                 fireCSharpEvent("gridClick", $(this).attr('id'));
             });
+
+            setTimeout(loadNextThumbnail, 100);
         });
         function fireCSharpEvent(eventName, eventData) {
 
             var event = new MessageEvent(eventName, { 'view': window, 'bubbles': true, 'cancelable': true, 'data': eventData });
             document.dispatchEvent(event);
         }
+
+        function loadNextThumbnail() {
```

Comment:
Needs a comment. I think it would be helpful to state (a) that the thumb-src attribute is created by the C# that generates the thumbnail-list html (is that right), and (b) the purpose of loading one thumbnail per 100s. (Is this still needed now you figured out about restricting our server to 4 threads?) BTW should we settle on 4 threads, or do something to figure out how many cpus we have and use that many?